### PR TITLE
250 Create all endpoints for organization 

### LINKED ISSUE #250: [Feat] Create all endpoints for organization
Make a file in `routes` folder with the name `organization.js`
make a URL as `organization/create`, `organization/list`, `organization/update`, `organization/delete`
this will be for all request with parameters that will do operations on organization based on filters on attribution

### PR DESCRIPTION
**Pull request by hricha11**
## Description
Create all endpoints for organization
## Fixes
fixes #250 
## Checklist

- [x] Code follows project's style guidelines.
- [x] Changes are documented appropriately.

### CODE REVIEW

File: controller/organization.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import {
+    addNewOrganization, deleteOrganizationById, updateOrganizationById, getOrganization,
```
Comment (sumitst05) [inline controller/organization.js:2]: The function to get organization as defined in `services/organization.js` is named `getOrganizations` and not `getOrganization`

File: controller/organization.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    } catch (error) {
+      logger.error("Error while inserting", error);
+      res.status(500);
+      res.json({ err: "Error while inserting in DB" });
+    }
+  }
+  
+  async function showOrganization(req, res) {
+    try {
+      const organization = await getOrganization(req.query);
```
Comment (sumitst05) [inline controller/organization.js:48]: change `getOrganization` to `getOrganizations` here as well

**Review by sumitst05 — CHANGES_REQUESTED** [review]
Changes requested.

File: controller/organization.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    } catch (error) {
+      logger.error("Error while inserting", error);
+      res.status(500);
+      res.json({ err: "Error while inserting in DB" });
+    }
+  }
+  async function deleteOrganization(req, res) {
+    const { organizationId } = req.params;
+    try {
+      await deleteOrganizationById(accredationId);
```
Comment (sumitst05) [inline controller/organization.js:22]: change `accreditationId` to `organizationId`

File: services/organization.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  const updated = await Organization.update({ _id: id }, data);
+  if (updated) {
+    return updated;
+  }
+  throw new databaseError.DataEntryError("Organization");
+}
+
+export default {
+  deleteOrganizationById, addNewOrganization, updateOrganizationById,
+};
```
Comment (kskarthik5) [inline services/organization.js:40]: add getOrganizations here

**Review by kskarthik5 — CHANGES_REQUESTED** [review]

**Review by TejasNair9977 — APPROVED** [review]
LGTM

**Review by TejasNair9977 — APPROVED** [review]
LGTM

**Pull request merged**